projects/common/src/utilities/validators/form-validators.ts

```diff
@@ -1,10 +1,16 @@
 import { AbstractControl } from '@angular/forms';
+import { isEmpty, isNil } from 'lodash-es';
 import { getStringsFromCommaSeparatedList } from '../formatters/string/string-formatter';
 import { areEmailAddressesValid } from './email-validator';
 
 // tslint:disable-next-line: no-namespace
 export namespace CustomFormValidators {
   export const commaSeparatedEmails = (control: AbstractControl) => {
+    if (isNil(control.value) || isEmpty(control.value)) {
+      // tslint:disable: no-null-keyword
```

Comment:
Use `disable-next-line` instead.

Reply:
Ooo. Good catch. We should always be using next-line. I copied/pasted from below. I'll update both.